From 6d7235490b2e5a34f8fe0d34be39c5780ef64653 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sun, 23 Jun 2024 10:08:25 -0700
Subject: [PATCH] ==== docs fixup for corbet ====

Subject: minor document fixups

This patch series is for minor document fixups.  First five patches
clean up and remove 'Other material' section of process/index in favor
of 'staging/' and better place for docs on the section.  A patch for
adding DAMON maintainer-profile reference on maintainers' handbook
follows.  And then the last patch adds 'hkml' as one of available email
tools on email-clients document.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/5lRqVNoY | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/5lRqVNoY

diff --git a/damon_meta_changes/5lRqVNoY b/damon_meta_changes/5lRqVNoY
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

